### ISSUE TITLE
Stop repeating the same pictures when there are new ones to show

### ISSUE THREAD

**Issue by sabbbi** [b-lam/Resplash#167]

**Comment by b-lam** [comment]
Hi Sabrina, thanks for opening up this issue. I just wanted to get some more details about what you are experiencing. Is this a bug or a feature request? Are pictures repeating as you scroll through the list or are you talking about the same photos being shown between sessions on the same day?

**Comment by sabbbi** [comment]
Hi,

I'm using resplash as a wallpaper changer on my pixel 7 pro. I use
collections with a lot of photos to randomly pick from but I keep seeing
the samen pictures on my background. Like the algorithm keeps forgetting
that some have already been used.

Does this clarify it?

Op ma 20 feb. 2023 02:42 schreef Brandon Lam ***@***.***>:

**Comment by b-lam** [comment]
@sabbbi Yes, thanks for the clarification! One possible explanation for this is using multiple collections but having one with fewer images. This can skew the randomness because first a random collection is selected, then a random photo is selected from that collection. So if you have 2 collections selected for auto wallpaper and one has 5 photos and the other has 100, you are much more likely to see a photo from the collection with 5 photos. Let me know if this sounds like your situation.
